apps/judicial-system/web/src/utils/hooks/useS3Upload/useS3Upload.ts

```diff
@@ -281,11 +287,14 @@ const useS3Upload = (caseId: string) => {
             // We need to set the id so we are able to delete the file later
             newFileId,
           )
+
+          return true
```

Comment:
Returning from the loop means that we only try to upload the first file. The rest are ignored.